packages/tools/devtools/devtools-view/src/components/RefreshButton.tsx

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import { Button, Tooltip } from "@fluentui/react-components";
+import { ArrowSync24Regular } from "@fluentui/react-icons";
+import { GetContainerList } from "@fluidframework/devtools-core/internal";
+import { useLogger } from "../TelemetryUtils.js";
+import { useMessageRelay } from "../MessageRelayContext.js";
+
+/**
+ * Message sent to the webpage to query for the full container list.
+ */
+const getContainerListMessage = GetContainerList.createMessage();
+
+/**
+ * A refresh button to retrieve the latest list of containers.
+ */
+export function RefreshButton(): React.ReactElement {
```

Comment:
Giving this component its own file and a general name like "RefreshButton" implies that its somewhat general purpose. But it looks like this is still very much about refreshing the container list for the app. We should either make this general purpose, or (preferably) move it back to `Menu.ts` and make it private.